cli/dts/lib.deno.worker.d.ts

```diff
@@ -43,33 +54,93 @@ declare class WorkerNavigator {
 
 declare var navigator: WorkerNavigator;
 
+interface DedicatedWorkerGlobalScopeEventMap extends WorkerGlobalScope {
```

Comment:
I think this should extend `WorkerGlobalScopeEventMap` instead? Other than this, it looks good.